src/display/displayTemplateManager.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "displayCommon.h"
+
+namespace Templates {
+namespace Standard {
+#include "displayTemplateStandard.h"
+}
+
+namespace Minimal {
+#include "displayTemplateMinimal.h"
+}
+
+namespace TempOnly {
+#include "displayTemplateTempOnly.h"
+}
+
+namespace Scale {
+#include "displayTemplateScale.h"
+}
+} // namespace Templates
+
+class DisplayTemplateManager {
+    public:
+        enum Template {
```

Comment:
Hm, What I would do is not use namespaces but classes. You have a base `class Template` and then a `class Standard : public Template`. Then the manager instance does not need an additional `enum` but can directly say `currentPrintScreen = std::make_shared<T>()`